src/Tribe/Rewrite.php

```diff
@@ -637,7 +647,7 @@ static function ( $localized_slug ) {
 	}
 
 	/**
-	 * Returns a map relating localize matcher slugs to the corresponding query var.
+	 * Returns a map relating localized matcher slugs to the corresponding query var.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 85 characters

<sub>hash: 8e18d7f6323c116e6f4062c302bb4fe8</sub>